packages/taiko-client/driver/config.go

```diff
@@ -126,6 +127,11 @@ func NewConfigFromCliContext(c *cli.Context) (*Config, error) {
 		return nil, err
 	}
 
+	// Check if the P2P signer setup is empty.
+	if c.Uint64(flags.PreconfBlockServerPort.Name) > 0 && utils.IsNil(signerConfigs) {
+		return nil, errors.New("empty P2P signer setup")
```

Comment:
```suggestion
		return nil, errors.New("`--p2p.sequencer.key` flag is required for preconfirmation server")
```

Will this error message make more sense to users?